src/main/java/com/maxmind/minfraud/request/CreditCard.java

```diff
@@ -149,6 +152,24 @@ private CreditCard(CreditCard.Builder builder) {
             return this;
         }
 
+        /**
+         * @param wasSuccessful An indication of whether or not the outcome of
+         *                      3D-Secure verification (e.g. Safekey,
+         *                      SecureCode, Verified by Visa)  was successful,
+         *                      as provided by the end user. {@code true} if
+         *                      customer verification was successful, or
+         *                      {@code false} if the customer failed
+         *                      verification. If 3-D Secure verification was not
+         *                      used, was unavailable, or resulted in another
+         *                      outcome other than success or failure, do not
+         *                      use this method.
+         * @return The builder object.
+         */
+        public CreditCard.Builder was3DSecureSuccessful(Boolean wasSuccessful) {
```

Comment:
Java itself is inconsistent on how to handle acronyms/initialisms. All our Java libraries follow the pattern that only the first letter should be capitalized, so `3d` in this case.